core/translate.c

```diff
@@ -225,6 +248,14 @@ translate_walk_track(dcontext_t *tdcontext, instr_t *inst, translate_walk_t *wal
             walk->translation = instr_get_translation(inst);
             LOG(THREAD_GET, LOG_INTERP, 5, "%s: entering mangle region xl8=" PFX "\n",
                 __FUNCTION__, walk->translation);
+        } else if (IF_X86_ELSE(
```

Comment:
Do we know that AARCHXX doesn't have the same problem?

Reply:
Can't test, and some comment mentioned that ARCHXX has different assumptions about spills (might expose the more complex mangling case we just talked about!)